web/massastation/src/pages/Store/StoreSection/StoreSection.tsx

```diff
@@ -47,6 +47,8 @@ function StoreSection({
     );
   };
 
+  const sortedPlugins = plugins?.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
Last: can we have that in a classical function?

Reply:
ofc